tests/consumers/acero_consumer.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+import pyarrow as pa
+
+try:
+    import pyarrow.substrait as substrait
+except ImportError:
+    substrait = None
```

Comment:
I assume PyArrow built with Substrait is a requirement for this package or is it optional? 
If it is not optional we should probably have this just as `import pyarrow.substrait as substrat` or `import pyarrow.substrait as sbt`